src/tools/crawling_tools.py

```
@@ -991,31 +1216,122 @@ async def _process_code_examples(ctx: Context, crawl_results: List[Any], directo
     return code_examples_stored
 
 
-async def _store_in_neo4j(ctx: Context, directory_path: str) -> bool:
+async def _store_in_neo4j(ctx: Context, directory_path: str, crawl_results: List = None) -> int:
```

Ruff (0.12.2)

`_store_in_neo4j` is too complex (12 > 10)

(C901)